pkg/task/job.go

```diff
@@ -13,6 +13,8 @@ const (
 	DecompressTaskType
 	// TransferTaskType 中转任务
 	TransferTaskType
+	// RecycleTaskType 回收任务
+	RecycleTaskType
```

Comment:
the same as above

Reply:
This enum is saved in DTO.
And maybe `GetJobFromModel` will get confusing

Reply:
You are right, fiexed.